lib/rex/post/sql/ui/console/command_dispatcher/client.rb

```diff
@@ -120,6 +120,11 @@ def cmd_query_help
               def run_query(query)
                 begin
                   result = client.query(query)
+                rescue EOFError => e
+                  elog("Running query '#{query}' failed on session #{self.inspect}", error: e)
+                  session.alive = false
+                  session.client.interacting = false if session.client && session.client.respond_to?(:interacting)
+                  return { status: :error, result: { errors: [e] } }
                 rescue ::RuntimeError, ::StandardError => e
```

Comment:
Not sure why we're catching RuntimeError explicitly here, it looks like it's a StandardError already

```
3.3.0 :001 > RuntimeError.ancestors
 => [RuntimeError, StandardError, Exception, Object, PP::ObjectMixin, Kernel, BasicObject] 
```